docs/modules/behaviors/DATA_BEHAVIOR.md

```diff
@@ -807,9 +808,4 @@ successEventPayload = {
   * "behaviors.behaviorAlias:[event name]" - arbitrary event triggered by another behavior on this view.
   * "behaviors.behaviorAlias.[property name]:[event name]" - arbitrary event triggered by another behavior's property on this view.
   * "behaviors.behaviorAlias.data:[event name]" - arbitrary event triggered by another behavior's data property on this view (specific example of `behaviorAlias.[property name]:[event name]`).
-  * { '[event name]': < object (or function returning an object) that the event is triggered on > } - arbitrary "event" triggered on the supplied object.
-
-
-
-
-
+  * { '[event name]': < object (or function returning an object) that the event is triggered on > } - arbitrary "event" triggered on the supplied object.
```

Comment:
I feel like this comment could get a little love. I read it a couple times and still wasn't sure what its purpose was. Is it triggering [event name] on the object, or listening to [event name] on the object, and then doing what when it is triggered?

Reply:
see #341